pkg/rng/entropy.go

```diff
@@ -6,12 +6,11 @@ package rng
 
 import (
 	"errors"
+	"fmt"
```

Comment:
portions of this seem very linux-specifici. Might be best to have entropy.go, and then entropy_linux.go, and entropy_other.go so we have a change of getting it going on all kernels.